### PR TITLE
Routingtable as label

### PR DESCRIPTION
**Pull request by douardda**
Make the generated routing table "referencable" using the 'routingtable' label.

### CODE REVIEW

**Comment by douardda** [comment]
@dahlia yup, but which version are we targeting? 1.7.1 or 1.8?

**Comment by dahlia** [comment]
@douardda As its target branch is master, the target version should be 1.8.0.

**Comment by douardda** [comment]
@dahlia ok I think it should be good now.

File: doc/conf.py [hunk 1]
```diff
@@ -69,9 +69,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '1.7'
+version = '1.8'
 # The full version, including alpha/beta/rc tags.
-release = '1.7.0'
+release = '1.9.0'
```
Comment (dahlia) [inline doc/conf.py]: It seems typo.  It should be 1.8.0.

**Comment by dahlia** [comment]
Thank you!

**Pull request merged**